LICENSE.txt

```diff
@@ -0,0 +1,13 @@
+Copyright 2004-2016 Castle Project - http://www.castleproject.org/
```

Comment:
@alinapopa great to know NuGet isn't broken, just the new tooling has a defect we can easily work around rather than renaming the file. Should we rename it back in the branch in Windsor?